### ISSUE TITLE
Support for different internal socket host

### ISSUE THREAD

**Issue by Wiejeben** [MatviiB/notifier#13]
I would like to see the possibility to configure a different internal socket hosts for internal communication.

Specifically just have the possibility to set a different host in the browser <script> tag generated by `@include('notifier::connect')`.

Thanks!

**Comment by MatviiB** [comment]
You can configure connection in your .env file
Look [config ](https://github.com/MatviiB/notifier/blob/master/src/config/notifier.php)

**Comment by Wiejeben** [comment]
That configures the socket host for both the external and internal communication. I need the `@include('notifier::connect')` `SOCKET_HOST` to be different than the `SOCKET_HOST` used for `event(new Notify(...))` and `php artisan notifier:init`.

**Comment by MatviiB** [comment]
why do you need different?

**Comment by Wiejeben** [comment]
In my case, I run my stuff inside Docker, this creates a sandbox between each application. For example, PHP and NGINX are fully separated, but only communicate with each other on the internal network.

I need to run `php artisan notifier:init` inside a separate container. In order to allow this, I would want the internal socket communicatie to be handled via a different host, and let the visitors webbrowser connect to the publicly defined host, for example `website.com`. 

There are many ways I could put this I guess. But to me, internally, the external host isn't accessibly, internal communication can only be handled internally via a locally available hostname, like `localhost`, or the hostname of a different Docker instance.

P.S. I have tried to fork this repo but was unable to set it up locally as a path repository. But since this is a small change (obviously of no internal host is defined, fallback to the regular `SOCKET_HOST`), I hope I hope I described the issue clear enough.
